util/extract-metadata.py

```diff
@@ -1,242 +1,227 @@
 #!/usr/bin/env python3
 
-__author__ = 'cjm'
-
 import argparse
-import logging
-
-#from yaml import load, dump
-#from yaml import Loader, Dumper
+import sys
 import yaml
 
-def main():
+__author__ = 'cjm'
 
-    parser = argparse.ArgumentParser(description='OBO'
-                                                 'Helper utils for OBO',
-                                     formatter_class=argparse.RawTextHelpFormatter)
 
+def main():
+  parser = argparse.ArgumentParser(description='Helper utils for OBO',
+                                   formatter_class=argparse.RawTextHelpFormatter)
+  subparsers = parser.add_subparsers(dest='subcommand', help='sub-command help')
 
-    subparsers = parser.add_subparsers(dest='subcommand', help='sub-command help')
-    
-    # SUBCOMMAND
-    parser_n = subparsers.add_parser('validate', help='validate yml inside md')
-    ##parser_n.add_argument('-d', '--depth', type=int, help='number of hops')
-    parser_n.set_defaults(function=validate_markdown)
-    parser_n.add_argument('files',nargs='*')
+  # SUBCOMMAND
+  parser_n = subparsers.add_parser('validate', help='validate yaml inside md')
+  # parser_n.add_argument('-d', '--depth', type=int, help='number of hops')
+  parser_n.set_defaults(function=validate_markdown)
+  parser_n.add_argument('files', nargs='*')
 
-    # SUBCOMMAND
-    parser_n = subparsers.add_parser('concat', help='concat ontology yamls')
-    parser_n.add_argument('-i', '--include', help='yml file to include for header')
-    parser_n.add_argument('-o', '--output', help='output yaml')
-    parser_n.set_defaults(function=concat_ont_yaml)
-    parser_n.add_argument('files',nargs='*')
+  # SUBCOMMAND
+  parser_n = subparsers.add_parser('concat', help='concat ontology yamls')
+  parser_n.add_argument('-i', '--include', help='yaml file to include for header')
+  parser_n.add_argument('-o', '--output', help='output yaml file')
+  parser_n.set_defaults(function=concat_ont_yaml)
+  parser_n.add_argument('files', nargs='*')
 
-    # SUBCOMMAND
-    parser_n = subparsers.add_parser('concat-principles', help='concat principles yamls')
-    parser_n.add_argument('-i', '--include', help='yml file to include for header')
-    parser_n.add_argument('-o', '--output', help='output yaml')
-    parser_n.set_defaults(function=concat_principles_yaml)
-    parser_n.add_argument('files',nargs='*')
+  # SUBCOMMAND
+  parser_n = subparsers.add_parser('concat-principles', help='concat principles yamls')
+  parser_n.add_argument('-i', '--include', help='yaml file to include for header')
+  parser_n.add_argument('-o', '--output', help='output yaml')
+  parser_n.set_defaults(function=concat_principles_yaml)
+  parser_n.add_argument('files', nargs='*')
 
-    args = parser.parse_args()
+  args = parser.parse_args()
 
-    func = args.function
-    func(args)
+  func = args.function
+  func(args)
 
 
 def validate_markdown(args):
```

Comment:
This job is now done better by the `validate-metadata.py` script and schemas. It isn't used in the `Makefile`. I think we should remove this function and subcommand parser.

Reply:
Hi James (and all), thanks for having a look. FYI you probably already know this, but you can hide whitespace changes temporarily while reviewing the commit by clicking on the little settings (gear) icon on the Files Changed tab.

Reply:
Thanks @lmcmicu. Actually I didn't know about that option. You learn something new every day.